LAUNCH Test Project/Assets/Scripts/CameraPos.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace CameraToolkit
+{
+	public class CameraPos : MonoBehaviour
+	{
+        public Interpolator optionalInterpolator;
+        public Button forwardButton;
+        public Button backwardsButton;
+        public int forwardDistance;
+        public int backwardsDistance;
+        private void AddPosition(Vector3 pos)
+        {
+            if (optionalInterpolator != null)
+            {
+                optionalInterpolator.targetPosition += pos;
+            }
+            else
+            {
+                transform.position += pos;
+            }
+            }
+        ///Vector3 endPos;
+        ///Vector3 startPos;
+        ///float lerpMult;
+        private void Start()
+        {
+            Button btn1 = forwardButton;
+            btn1.onClick.AddListener(CamForward);
+
+            Button btn2 = backwardsButton;
+            btn2.onClick.AddListener(CamBackwards);
+        }
+
+        void CamForward()
```

Comment:
Add "private" access modifier, it defaults to that anyways but good to be explicit